src/main/java/frc/robot/subsystems/AprilTagDetection.java

```diff
@@ -11,9 +11,9 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import java.util.ArrayList;
 
-public class AprilTagFinder extends SubsystemBase {
```

Comment:
So I think this is mis-named. The subsystem is not a detection. The detection concept represents an individual instance/result of the detection process. The subsystem would be a "AprilTagDetector" It is the thing that creates and maintains detections, right?

Reply:
Yeah that would make more sense. I'll close out this pull request and rename it at the meeting tonight.